bitcoin/src/blockdata/constants.rs

```diff
@@ -47,7 +47,10 @@ pub const MAX_STACK_ELEMENT_SIZE: usize = 520;
 /// How may blocks between halvings.
 pub const SUBSIDY_HALVING_INTERVAL: u32 = 210_000;
 /// Maximum allowed value for an integer in Script.
-#[deprecated(since = "TBD", note = "This constant has ambiguous semantics. Please carefully check your intended use-case and define a new constant reflecting that.")]
+#[deprecated(
+    since = "TBD",
+    note = "This constant has ambiguous semantics. Please carefully check your intended use-case and define a new constant reflecting that."
+)]
```

Comment:
I give up, i just passed on all commits running "just fmt" to use yall tools for formating but it keeps changing...

Reply:
Yeah, I'm not sure whats going on, your editor should pick up the config in `rustfmt.toml`.

Reply:
Ill try better